frontend/test/metabase/scenarios/organization/timelines-question.cy.spec.js

```diff
@@ -256,6 +256,15 @@ describe("scenarios > organization > timelines > question", () => {
 
       cy.findByText("Visualization").should("be.visible");
       cy.findByLabelText("star icon").should("be.visible");
+
+      // should hide individual events from chart if hidden in sidebar
+      cy.icon("calendar").click();
```

Comment:
This doesn't seem to have anything in common with the original intent of this test. Could we move it to a separate test?

Reply:
@ranquild We can, but at the cost of making e2e tests slower due to setup for each run.

@nemanjaglumac has been in favor of making longer individual e2e test cases.

Maybe we can add something to the title of the test to mention hiding?

Reply:
I'll let @nemanjaglumac decide what assertions should be combined in one test.

Reply:
@ranquild @nemanjaglumac 

With the naming change from "hidden" to "visible" in all the props and functions, we have to do work to ensure a new event is visible upon creation.

We'd better test that, and we are.

Now the e2e test covers enough ground to warrant its own block, see if you disagree.

Reply:
Tests have changed enough from first comment to warrant other conversations.